pkgs/tools/system/pciutils/default.nix

```diff
@@ -14,8 +14,10 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ zlib kmod which ] ++
-    lib.optional stdenv.hostPlatform.isDarwin IOKit;
+  buildInputs = [ zlib ]
+    ++ lib.optional stdenv.hostPlatform.isLinux kmod
+    ++ [ which ]
```

Comment:
why is this not in the same list as zlib

Reply:
The goal is to prevent any rebuilds on `linux`. `pciutils` gets pulled in by `systemd` and quite a few other packages. Might be a bit too disruptive for `staging-next`. To make it a no-op I preserved the original ordering. It' makes it a bit more verbose.

Reply:
Since we are targeting staging-next anyway we trigger some rebuilds or just target staging.

Reply:
Target reordering introduces 12K rebuilds:

```
$  ./maintainers/scripts/rebuild-amount.sh HEAD^
Estimating rebuild amount by counting changed Hydra jobs (parallel=unset).
      1 pkgs-lib-tests
      3 x86_64-darwin
  12621 x86_64-linux
```

Our whole queue right now is 15K builds. I'll retarget against `staging`.